chapter_linear-networks/softmax-regression-scratch_vn.md

```diff
@@ -220,7 +220,7 @@ def cross_entropy(y_hat, y):
 ## Classification Accuracy
 -->
 
-## *dịch tiêu đề phía trên*
+Độ chính xác cho bài toán phân loại
```

Comment:
Độ chính xác nghe giống precision hơn là accuracy (https://blog.minitab.com/blog/real-world-quality-improvement/accuracy-vs-precision-whats-the-difference). Tính đúng đắn thì lại dễ nhầm với truthfulness.  Thuật ngữ này nên để thảo luận.

Reply:
dạ em nghĩ dễ xảy ra chiều ngược lại hơn, tức nếu dịch precision là chính xác sẽ hiểu lầm là accuracy ạ.

Reply:
Như anh @tiepvupsu có comment là nào giờ nhiều sách dịch `accuracy` là `độ chính xác` rồi nên để vậy hợp lý hơn.